packages/nimble/inst/tests/test-macros.R

```diff
@@ -0,0 +1,222 @@
+source(system.file(file.path('tests', 'test_utils.R'), package = 'nimble'))
+
+nimbleOptions(enableModelMacros = TRUE)
+
+context("Testing model macros")
+
+test_that('Macro expansion 1',
+{
+    ## This converts a ~ testMacro1(b)
+    ## to inputs as (stoch = TRUE, LHS = a, b)
+    testMacro1 <- nimble:::model_macro_builder(
```

Comment:
nit: Since you've encapsulated these macros in a `test_that('', {})`, it is clearer to avoid the numeric suffix in `testMacro1`, `testMacro2`, etc. That way it is easier to copy-and-modify tests and to remove or reorder existing tests without breaking some vestigial numerical ordering.